src/screens/Messages/ChatList.tsx

```diff
@@ -165,7 +165,9 @@ export function MessagesScreen({navigation, route}: Props) {
                       color="secondary"
                       variant="solid"
                       onPress={() => refetch()}>
-                      <ButtonText>Retry</ButtonText>
+                      <ButtonText>
+                        <Trans>Retry</Trans>
```

Comment:
I think this should be reviewed separately as a PR, rather than included in the Danish localization.

Reply:
Done: #7243